src/modules/Dropdown/Dropdown.js

```diff
@@ -489,7 +489,7 @@ export default class Dropdown extends Component {
     const { closeOnChange, multiple } = this.props
     const shouldClose = _.isUndefined(closeOnChange) ? !multiple : closeOnChange
 
-    if (shouldClose) this.close(e)
+    if (shouldClose) this.close(e, () => {})
```

Comment:
cruft?

Reply:
This is on purpose.
`callback` the 2nd argument in `close` method has default value.
But not using default value in this case.

```
close = (e, callback = this.handleClose) => {
    const { open } = this.state
    debug('close()', { open })

    if (open) {
      _.invoke(this.props, 'onClose', e, this.props)
      this.trySetState({ open: false }, callback)
    }
  }
```